website/content/api-docs/secret/databases/couchbase.mdx

```diff
@@ -46,6 +46,9 @@ has a number of parameters to further configure a connection.
 - `bucket_name` `(string: "")` - Required for Couchbase versions prior to 6.5.0. This
   is only used to verify vault's connection to the server.
 
+- `username_template` `(string: "")` - [Template](/docs/concepts/username-templating) describing how
+  dynamic usernames are generated. Defaults to the current behavior.
```

Comment:
Could we add an example of the format instead of referencing "the current behavior"? For someone unfamiliar, this doesn't give them much information and will make less sense over time

Reply:
Alternative to describing/showing the current behavior for each instance of `username_template` doc, we could add a small section to the `Username Templating` doc page that explains to the effect of "if the engine you're using doesn't support `username_templating`, this is the basic format you'll get with usernames: <example>"

So we define it in one spot, and people who are otherwise just reading the templating docs understand the basic default

Reply:
Re: adding the default templates here: I am hesitant to add them here as 1) they're fairly long, and 2) we never specified them previously and I don't want to imply that those defaults can't change. Maybe that's a pipe dream given Hyrum's law though. I'll look into it again and see if there's something reasonable we can do here.

Re: option 2: there isn't a single default for the databases so I don't think it makes sense to put them in a central location. Each DB is different with little overlap other than the most basic of structure.